src/main/java/uk/gov/di/ipv/service/ConfigurationService.java

```diff
@@ -29,9 +29,20 @@ public String getAuthCodesTableName() {
         return System.getenv("AUTH_CODES_TABLE_NAME");
     }
 
-    public CredentialIssuers getCredentialIssuers() {
-        return CredentialIssuerLoader.loadCredentialIssuers(
-                ssmProvider.get(System.getenv("CREDENTIAL_ISSUER_CONFIG_PARAMETER_STORE_KEY")));
+    public CredentialIssuers getCredentialIssuers(CredentialIssuers credentialIssuers) {
+
+        String credentialIssuerConfigBase64 =
+                ssmProvider.get(System.getenv("CREDENTIAL_ISSUER_CONFIG_PARAMETER_STORE_KEY"));
+
+        if (credentialIssuers == null
+                || !credentialIssuerConfigBase64.equals(credentialIssuers.getSource())) {
```

Comment:
It might be nice to move this source checking logic to a method on the `CredentialIssuers` class. That way we could do something like: 
`if (credentialIssuers == null || credentialIssuers.fromDifferentSource(credentialIssuerConfigBase64))`

Reply:
done